### PR TITLE
Update README & Adds contributing section

### PR DESCRIPTION
**Pull request by villander**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,25 +1,46 @@
-# ember-engines.com [![Build Status](https://travis-ci.org/ember-engines/ember-engines.com.svg?branch=master)](https://travis-ci.org/ember-engines/ember-engines.com)
+## Ember Engines guides
 
-This repo contains the Ember application for [ember-engines.com](http://ember-engines.com), the official documentation website for ember-engines.
+[![Build Status](https://travis-ci.org/ember-engines/ember-engines.com.svg?branch=master)](https://travis-ci.org/ember-engines/ember-engines.com)
 
-## Installation
+This repository holds the guides and tutorials for the [Ember Engines](https://github.com/ember-engines/ember-engines), a powerful addon that allow multiple logical applications to be composed together into a single application from the user's perspective.
```
Comment (dgeb) [inline README.md]:
```suggestion
This repository holds the guides and tutorials for [Ember Engines](https://github.com/ember-engines/ember-engines), a powerful addon that allow multiple logical applications to be composed together into a single application from the user's perspective.
```

File: README.md [hunk 2]
```diff
@@ -1,25 +1,46 @@
...
-```
+## Prerequisites
 
-## Development
+You will need the following things properly installed on your computer.
 
+* [Git](https://git-scm.com/)
+* [Node.js](https://nodejs.org/) (with npm)
+* [Ember CLI](https://ember-cli.com/)
+* [Google Chrome](https://google.com/chrome/)
```
Comment (dgeb) [inline README.md]: Is Chrome really required?

**Review by dgeb — CHANGES_REQUESTED** [review]
Looks great @villander! Just needs a couple tweaks AFAICT.

File: README.md (hunk 1 above)
Comment (villander) [reply README.md]: fixed!

File: README.md (hunk 2 above)
Comment (villander) [reply README.md]: No, you're right. fixed!

**Review by dgeb — APPROVED** [review]

**Pull request merged**